config/app/src/main/java/com/google/samples/quickstart/config/kotlin/MainActivity.kt

```diff
@@ -81,8 +82,8 @@ class MainActivity : AppCompatActivity() {
         // [START get_config_values]
         val welcomeMessage = remoteConfig[WELCOME_MESSAGE_KEY].asString()
         // [END get_config_values]
-        welcomeTextView.isAllCaps = remoteConfig[WELCOME_MESSAGE_CAPS_KEY].asBoolean()
-        welcomeTextView.text = welcomeMessage
+        binding.welcomeTextView.isAllCaps = remoteConfig[WELCOME_MESSAGE_CAPS_KEY].asBoolean()
+        binding.welcomeTextView.text = welcomeMessage
```

Comment:
nit: we could use Kotlin's `with` function here
```suggestion
        with (binding) {
            welcomeTextView.isAllCaps = remoteConfig[WELCOME_MESSAGE_CAPS_KEY].asBoolean()
            welcomeTextView.text = welcomeMessage
        }
```

Reply:
Is there a reason to prefer this style?  Is it safer in some way?  I always like things that are less indented personally but I want to do The Kotlin Way

Reply:
Doesn't really provide any additional safety, I see it as a way of not having to repeat "binding" in every line of code.